app/views/todos/_form.html.haml

```diff
@@ -0,0 +1,12 @@
+#todo
+  = simple_form_for @todo, remote: true do |f|
+    %p=f.input :title
+    %p=f.input :due, as: :date,
+                    start_year: Time.now.year - 1,
+                    end_year: Time.now.year,
+                    start_month: Time.now.month - 1,
+                    end_month: Time.now.month,
+                    order: [:month, :day, :year]
+    %p=f.input :completed, as: :boolean, checked_value: true, unchecked_value: false
```

Comment:
Line is too long. [84/80]<br>The = symbol should have one space separating it from code